### PR TITLE
[Tests-Only] Added update password UI tests to check password requirements

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
With this PR
- Added password update UI tests cases to check different password requirements
## Related Issue
- Fixes https://github.com/owncloud/password_policy/issues/126
- Fixes https://github.com/owncloud/password_policy/issues/315

### CODE REVIEW

File: tests/acceptance/features/webUIPasswordExpire/userPasswordExpire.feature [hunk 1]
```diff
@@ -15,3 +15,28 @@ Feature: expire user's password using the occ command
     Then the user should be redirected to a webUI page with the title "%productname%"
     When user "Alice" enters the current password, chooses a new password "newpassword" and confirms it using the webUI
     Then the user should be redirected to a webUI page with the title "Files - %productname%"
+
+  Scenario: password update with wrong old password
+    Given the administrator has enabled the days until user password expires user password policy
+    And the administrator has expired the password of user "Alice"
+    Given user "Alice" has logged in with expired password using the webUI
```
Comment (Talank) [inline tests/acceptance/features/webUIPasswordExpire/userPasswordExpire.feature:22]:
```suggestion
    And user "Alice" has logged in with expired password using the webUI
```

File: tests/acceptance/features/lib/UpdatePasswordPage.php [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+/**
+ * PageObject for the password policy settings Page
+ *
+ */
+class UpdatePasswordPage extends OwncloudPage {
+	/**
+	 * @var string path
+	 */
+	protected $path = '/index.php/apps/password_policy/update_password?redirect_url=/coreAgain/index.php/apps/files/';
```
Comment (Talank) [inline tests/acceptance/features/lib/UpdatePasswordPage.php:37]: Why protected? :thinking:

Comment (kiranparajuli589) [reply tests/acceptance/features/lib/UpdatePasswordPage.php:37]: IDK too...just copied and pasted it from `settingsPage` :thinking: Implemented in the same way for `$path` in other `pageObjects`.

Comment (phil-davis) [reply tests/acceptance/features/lib/UpdatePasswordPage.php:37]: behat-page-object-extension `Page.php` has:
```
abstract class Page extends DocumentElement implements PageObject
{
    /**
     * @var string|null
     */
    protected $path = null;
...
```
so this is using the same variable scope as in the parent (grandparent) that it is overriding.

Comment (phil-davis) [reply tests/acceptance/features/lib/UpdatePasswordPage.php:37]: https://www.php.net/manual/en/language.oop5.visibility.php - the parent abstract class does not want to make the value public, but expects the implementing class to be able to set, see and use it. So private also would not work. That leaves `protected`

**Review by phil-davis — APPROVED** [review]

**Pull request merged**